### PR TITLE
Added a main and functions file

### PR DESCRIPTION
**Pull request by ns-1998**
Added basic files to project body.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+README Basic code
+================
+
+Basic operation over two number like add, sub, mul and div.
```
Comment (1911niraj) [inline README.md:4]: Please clear the operation name instead of using short forms.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+EXAMPLE
```
Comment (1911niraj) [inline README.md:6]: use '##' before example.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+======
+
+Take two numbers 2 and 3.
+Then sum of 2 and 3 is 5.
+Then sub of 2 and 3 is -1.
+Then mul of 2 and 3 is 6.
+Then div of 2 and 3 is 0.
```
Comment (1911niraj) [inline README.md:13]: modify the name (please use full form).

File: functions.c [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+
```
Comment (1911niraj) [inline functions.c:2]: Add basic comment before functions and doxygen format.

File: main.c [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int main()
+{
+	int a=2, b=3;
+	printf("Sum of %d and %d is %d.\n",a,b,sum(a,b));
+	printf("Sub of %d and %d is %d.\n",a,b,sub(a,b));
+	printf("Mul of %d and %d is %d.\n",a,b,mul(a,b));
+	printf("Div of %d and %d is %d.\n",a,b,div(a,b));
+}
```
Comment (1911niraj) [inline main.c:10]: Add return at end of the function.

File: main.c [hunk 6]
```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int main()
```
Comment (1911niraj) [inline main.c:3]: specify what function is doing.

File: main.c [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+{
+	int a=2, b=3;
```
Comment (1911niraj) [inline main.c:5]: make user input numbers.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+README Basic code
```
Comment (1911niraj) [inline README.md:1]: typo : "README of Basic code"

File: functions.c [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+int sub(int a, int b)
+{
+	return a-b;
+}
+
+int mul(int a, int b)
+{
+	return a-b;
```
Comment (1911niraj) [inline functions.c:15]: function return subtracted value in multiplication function.

File: functions.c [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+int div(int a, int b)
+{
+	return a-b;
```
Comment (1911niraj) [inline functions.c:20]: function return subtracted value in division function.

Comment (1911niraj) [inline functions.c:20]: Handle "0" divisor case by printing error message and returning 0.

**Comment by ns-1998** [comment]
Successfully updated comments in new commit.

**Review by 1911niraj — APPROVED** [review]

**Pull request merged**